packages/front-end/components/Metrics/FactMetricTypeName.tsx

```diff
@@ -0,0 +1,16 @@
+import { FactMetricType } from "back-end/types/fact-table";
+import { capitalizeFirstLetter } from "@/services/utils";
+
+export default function FactMetricTypeName({ type }: { type: FactMetricType }) {
```

Comment:
question: Should we make this handle both types of Metrics by default?

Also wdyt about `MetricTypeDisplayName` to emphasize that this is the formatting that we want to show users vs internal names.

Reply:
Renamed it. TBH I don't really want to spend time thinking about legacy metrics at all, so I didn't want to try and cover all those use cases.